crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -1181,18 +1181,23 @@ impl AuthorityPerEpochStore {
         &self,
         checkpoint: &CheckpointSequenceNumber,
     ) -> SuiResult<Option<Accumulator>> {
-        Ok(self.tables()?.state_hash_by_checkpoint.get(checkpoint)?)
+        Ok(self
```

Comment:
given that these return `SuiResult` already and that's not changing, what benefit is there to adding the `expect` here vs letting the higher layer decide?

Reply:
Since tables() is fallible, we have no choice but to handle that error higher up. But I've been on a mission to panic on all db errors right where they happen.